src/pages/ListScholarships.jsx

```diff
@@ -96,13 +96,19 @@ function ListScholarships() {
     } else pruneQueryParam(qParams.GRADES);
   }
 
+  if (majors !== undefined) {
+    if (!Array.isArray(majors) || majors.length === 0 || majors[0] === '')
```

Comment:
`majors[0] === ''` takes care of the case where `?majors[]=` is inputted in the URL